apps/internal/slog/logger_121_below.go

```diff
@@ -0,0 +1,35 @@
+//go:build !go1.21
+
+package slog
+
+import (
+	"context"
+)
+
+type Level int
+
+type Logger struct{}
+
+const (
+	LevelDebug Level = iota
+	LevelInfo
+	LevelWarn
+	LevelError
+)
+
+type NopHandler struct{}
```

Comment:
not a blocker but consider defining this once, in a separate file e.g. `nop_handler.go` with no build constraint